destination/writer/writer_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package writer
+
+import (
+	"context"
+	"testing"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/matryer/is"
+)
+
+func TestWriter_WriteFailNoPayload(t *testing.T) {
+	t.Parallel()
+
+	is := is.New(t)
+
+	w := Writer{}
+
+	record := sdk.Record{
+		Key: sdk.StructuredData{
+			"id": 1,
+		},
+	}
+
+	err := w.Write(context.Background(), record)
+	is.Equal(err, errNoPayload)
+}
```

Comment:
hm, maybe add some successful test?

Reply:
It'll be great, of course 👍 
I'll try to add the unit test for the `New` method.